Cython/Compiler/ExprNodes.py

```diff
@@ -189,20 +189,31 @@ def infer_sequence_item_type(env, seq_node, index_node=None, seq_type=None):
 
 def get_exception_handler(exception_value):
     if exception_value is None:
-        return "__Pyx_CppExn2PyErr();"
+        return "__Pyx_CppExn2PyErr();", False
```

Comment:
It would be nice to have some kind of indication in this function what the `True`/`False` return values mean. At least a comment or docstring, but some named constant could also make it clearer. Or a named tuple?